include/inexor/vulkan-renderer/world/cube.hpp

```diff
@@ -92,10 +92,12 @@ class Cube : public std::enable_shared_from_this<Cube> {
     Cube(Cube &&rhs) noexcept;
     ~Cube() = default;
     Cube &operator=(Cube rhs);
+    Cube &operator=(Cube &&) = delete;
```

Comment:
Delete?

Reply:
Shouldn't it be `noexcept = default` because we provide the move constructor, right?

Reply:
Probably.

Reply:
Doing `= default` here wouldn't actually be correct. We could implement the move operator, although probably not in this PR?

Reply:
Yes, I will do this in another PR.
https://github.com/inexorgame/vulkan-renderer/issues/374.